staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/compile.go

```diff
@@ -160,7 +160,7 @@ func buildRequestType() *apiservercel.DeclType {
 	))
 }
 
-// CompilationResult represents a compiled ValidatingAdmissionPolicy validation expression.
+// CompilationResult represents a compiled cel validation expression.
```

Comment:
revert search/replace here

Reply:
For this one it's in the generic package now so this no longer has any references to ValidatingAdmissionPolicy so I don't know if it makes sense in this comment to revert.